src/components/common/DataUseVoteSummary/VoteResultContainer.js

```diff
@@ -0,0 +1,64 @@
+import VoteResultIcon from './VoteResultIcon';
+import VoteResultLabel from './VoteResultLabel';
+import { isEmpty } from 'lodash/fp';
+import { h, div } from 'react-hyperscript-helpers';
+
+const convertLabelToKey = (label) => {
+  return label.split(' ').join('-');
+};
+
+const determineUnanimousVoteResult = ({votes = []}) => {
+  const voteCount = votes.length;
+  if (isEmpty(votes) || voteCount < 1) {
+    return 'underReview';
+  }
+  let voteTally = {
+    true: 0,
+    false: 0,
+  };
+
+  votes.forEach((vote = {}) => {
+    voteTally[vote.vote] += 1;
+  });
+
+  if (voteTally.true === voteCount) {
+    return true;
+  } else if (voteTally.false === voteCount) {
+    return false;
+  } else if (voteTally.true + voteTally.false === voteCount) {
+    return 'mixed';
+  } else {
+    return 'underReview';
+  }
```

Comment:
Another way to get the true/false/mixed answer is to collect the distinct `vote` values of all `votes` into an array. If that array has a single `true` element, then it's true, similarly if there's a single `false` value, and if there are two elements in that array, it's `mixed`. I've used this strategy at some point on the back end. Your call on this, I'm perfectly fine with the current approach.

Reply:
There's an added complication with underReview, where the election is still ongoing (vote is undefined). So if there's two elements in the mix, it could be `[(True/False), (True/False)]` or it could be `[(True/False), undefined]`. Either way, you'll have to confirm it with an additional loop on the sub-array and then running if checks on the possible combinations.